### ISSUE TITLE
Mailing list gone?

### ISSUE THREAD

**Issue by strandwg** [NOAA-PMEL/PyFerret#143]
Is there still a mailing list for Ferret/PyFerret? Have I completely lost touch? Thanks!

**Comment by Xiaoyu0131** [comment]
I would ask the same question. 😅

Gary Strand ***@***.***>于2024年2月3日 周六上午1:52写道：

**Comment by ACManke** [comment]
No, nothing has changed. There were a few messages last fall, but indeed it's gone very quiet. (The archive page, browse-by-year) has all messages since 2019 in the category 2019-2020)
